admin-dev/themes/new-theme/js/components/entity-search-input.ts

```diff
@@ -71,26 +93,27 @@ export default class EntitySearchInput {
    *
    * @param values {array}
    */
-  setValue(values) {
+  setValue(values: JQuery): void {
     this.clearSelectedItems();
     if (!values || values.length <= 0) {
       return;
     }
 
-    values.each((value) => {
+    values.each((index: number, value: any) => {
       this.appendSelectedItem(value);
     });
   }
 
   /**
    * Build the AutoCompleteSearch component
    */
-  buildAutoCompleteSearch() {
+  buildAutoCompleteSearch(): void {
     const autoSearchConfig = {
       source: this.entityRemoteSource,
       dataLimit: this.options.dataLimit,
+      value: '',
       templates: {
-        suggestion: (entity) => {
+        suggestion: (entity: OptionsObject) => {
```

Comment:
What is the difference between this `OptionsObject` and `any` I don't know where this type comes from, is it from typeahead?

Reply:
OptionsObject comes from `global.d.ts` which inherit from FetchObject types as it's Record<string, any>, I did this to  avoid repeating Record<string, any>, if we wants to define it better later that would probably easier, except if we need to create an interface for any OptionsObject

Dunnow if it's mandatory, or maybe you wan't me to replace every OptionsObject and FetchObject into Record<string, any> ? While the best later would be to have an interface for every responses / options